docs/parameters.yaml

```
@@ -148,6 +148,74 @@ type: filename
 default: none
 components: ["*"]
 ---
+name: Logging.XrootdPssSetOptOrigin
+description: >-
+  Configures the XRootD client used by the proxy top communicate with the origin (we use to set debug level).
+  Values include: DebugLevel (0, 1, 2, 3)
+type: string
+default: DebugLevel 1
+components: ["cache"]
+---
+name: Logging.XrootdPssSetOptCache
+description: >-
+  Configures the XRootD client used by the proxy top communicate with the cache (we use to set debug level).
+  Values include: DebugLevel (0, 1, 2, 3)
+type: string
+default: DebugLevel 3
+components: ["cache"]
+---
+name: Logging.XrootdPssTrace
+description: >-
+  Logging level of pss within Xrootd configuration values include: all, on, debug
```

```suggestion
  Logging level of pss within Xrootd configuration values include: all, on, debug, off
```
("off" is not listed in the docs but the code seems to accept it:
https://github.com/xrootd/xrootd/blob/8f8498d66aa583c54c0875bb1cfe432f4be040f4/src/XrdOuc/XrdOucPsx.cc#L805 )